CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.12.3
+### Fixed
+- Fix npm audit #5999
+- fix: permission check for cloning board #5854
+- Fix npm audit #5721
+- fix: Avoid conflicts on deck attachments folder name #5704
+- Fix npm audit #5679
+- Fix npm audit #5604
+
+### Added
+- don't reset update time when no update was written to db #6035
+- Clarify config file not found exception message #5725
+
+### Other
+- Update nextcloud/ocp dependency #5997
+- Update nextcloud/ocp dependency #5987
+- Update nextcloud/ocp dependency #5960
+- Update nextcloud/ocp dependency #5884
+- Update nextcloud/ocp dependency #5743
+- Update nextcloud/ocp dependency #5678
+- Update nextcloud/ocp dependency #5657
```

Comment:
```suggestion
- don't reset update time when no update was written to db #6035
- Clarify config file not found exception message #5725
- fix: permission check for cloning board #5854
- fix: Avoid conflicts on deck attachments folder name #5704
- Update dependencies
```

Just a suggestion to make it a bit more readable and useful to admins checking it out